### DISCUSSION TITLE
PiPedal v1.3.61 Release

### DISCUSSION THREAD

**Discussion by rerdavies** [rerdavies/pipedal#237] [General]
Bug fixes:
- Media files are not extracted properly from preset and bank files.
- Tuner control displayed for 3rd party tuner plugins that have outputs ports of type units:hz

 complete set of release notes can be found [here](https://rerdavies.github.io/pipedal/ReleaseNotes.html).

---
The .asc file is a gpg digital signature for the .deb file. You can verify the signature using the following command:

    gpg --verify pipedal_1.3.61_arm64.deb.asc pipedal_1.3.61_arm64.deb

Files are signed with the following key:

&nbsp;&nbsp;&nbsp;&nbsp;[PiPedal Project &lt;rerdavies@gmail.com&gt; - 2D1F 39DB B1F8 1941 2B67  8F88 E9D7 081E 08E3 D85C ](https://rerdavies.github.io/pipedal/pipedal-public-key.asc)

<hr /><em>This discussion was created from the release <a href='https://github.com/rerdavies/pipedal/releases/tag/v1.3.61'>PiPedal v1.3.61 Release</a>.</em>

**Comment by maarthome** [comment]
Hi Robin,

Forgot something or .... ?

Cheers,

Maarten

**Comment by EE1977** [comment]
Hi Robin!! I can't find the .deb file. Also pipedal can't find any update after 1.3.60.

**Comment by rerdavies** [comment]
The .deb file has been posted now. 

(I need to figure out some way to automate the odd little dance github requires when pushing a release. :-( )
